packages/dev/docs/pages/contribute.mdx

```diff
@@ -8,6 +8,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License. -->
 
 import {Layout} from '@react-spectrum/docs';
+import Contributing from '../../../../CONTRIBUTING.md';
```

Comment:
hmh, this commit breaks the docs...
Is something wrong with my path or is transclusion not supported?

Reply:
Looks like the reason is
> No transformers found for "/home/circleci/react-spectrum/CONTRIBUTING.md"
so Parcel doesn't know what to do with the md file it would seem, I'm sure there are some npm packages that can do this for us

Reply:
Okay, then this should help
https://mdxjs.com/getting-started/parcel

Reply:
Turns out you have already a custom `.mdx` transformer at https://github.com/adobe/react-spectrum/tree/main/packages/dev/parcel-transformer-mdx-docs